README.md

```diff
@@ -50,92 +49,92 @@ We will use the Photo API Service we built earlier to service the SwiftPhotoApp
 import UIKit
 
 class ViewController: UIViewController {
-
-    @IBAction func refreshButton(sender: UIButton) {
+    
+    @IBAction func refButton(sender: UIButton) {
         
         // Set up the URL Object.
-        let url = NSURL(string: "http://play.minio.io:8080/PhotoAPIService-0.0.1-SNAPSHOT/minio/photoservice/list")
+        let url = URL(string: "http://play.minio.io/PhotoAPIService-0.0.1-SNAPSHOT/minio/photoservice/list")
```

Comment:
i think you should make sure this URL is where harsha will deploy. I am wondering if we left http and are only https now.